.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: python
 
+os: linux
+
+dist: xenial
```

Comment:
I think that implicit dist would be enough for this project. Because the CI does not affect by Linux dist of the execution environment, and `os` is not limited Linux.

Reply:
The idea was using linux as main os+dist and just include the mac os and windows at the end.